packages/mapper/src/mapper.ts

```diff
@@ -171,15 +203,24 @@ export default class Mapper {
     options: Document = {}
   ): Promise<BulkWriteResult> {
     const dbOptions: any = {};
-    this.messageBus.emit('metho:bulkWrite', collection._name, operations);
+    const db = collection._database._name;
+    const coll = collection._name;
+    this.messageBus.emit(
+      'mongosh:api-call',
+      {
+        method: 'bulkWrite',
+        class: 'Collection',
+        db, coll, arguments: { operations, options }
```

Comment:
do we want to log the whole of operations here, in fact?

Reply:
I'd say, let's emit those so we have them in the telemetry. We could add some filter where we log if it gets messy.

Again not super against to giving names there, but i'd really start with arguments being an array and see if we need more.

We probably can't generate the names, so if we want to move that to either the `compile-shell-api.js` or to a decorator we would lose them.